gps/deduce_test.go

```diff
@@ -31,51 +31,51 @@ func mkurl(s string) (u *url.URL) {
 var pathDeductionFixtures = map[string][]pathDeductionFixture{
```

Comment:
All the `sdboyer` -> `golang/dep` changes in this file will need to be flipped back - this test is all about root deduction, and it doesn't work if it's not pointing to a project root anymore 😄